driver/web/authorization_bbs_permission_policy.csv

```diff
@@ -0,0 +1 @@
+p, image, /content/bbs/image, (POST) , allow
```

Comment:
Let's the permission be tps_update_images. Also, what does "allow" mean?. Also, the come is bad formatted.